google-cloud-storage/src/test/java/com/google/cloud/storage/BucketInfoTest.java

```diff
@@ -72,10 +72,12 @@
               LifecycleAction.newDeleteAction(),
               LifecycleCondition.newBuilder().setAge(5).build()));
   private static final String INDEX_PAGE = "index.html";
+  private static final String PUBLIC_ACCESS_PREVENTION_ENFORCED = "enforced";
```

Comment:
@athakor can you add a system test even if it doesn't pass right now?

Reply:
Yes, we'll definitely need a system test here.

Reply:
Most functionality should work currently for allowlisted projects.

Reply:
@shaffeeullah @tritone I provided our  java CI project IDs for the allow list in a comment on the design doc, thank you.